README.md

```diff
@@ -8,46 +8,58 @@ Codebase for Sparkle, brought to you by Sparkle.
 
 Clone the repo and cd into it
 
-```
+```bash
 git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`)
+<!-- Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`) -->
 
-```
+Now, obtain the `.env` file for the environment (ask someone on the engineering team) and save it.
```

Comment:
Since this repo is also open-source, we want to make sure that the instructions are also useful to people outside of Sparkle as well.

Keeping the note about saving it as `.env.development.local` (the previous suggestion of `.env.staging` is wrong) is a good idea.

Potentially we could also link to https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env to help explain what `.env` files can be used/why/etc.

An empty `.env.example` file exists at https://github.com/sparkletown/sparkle/blob/staging/.env.example that shows the format it needs to be in. So perhaps saying something like about copying `.env.example` to `.env.development.local`, and then filling it in with the appropriate details. For Sparkle, they can get that off other engineers. For random public, the main required variables relate to the deployed firebase project, so they can get them from there (the what/how of deploying this for the first time is probably out of scope (OOS) of this section of the README, so don't worry going into detailed specifics there)

Reply:
Since we will pretty much always use the same environment for local dev, we probably don't need to mention "for the environment" here either.